nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/TestTreeIndexing.java

```diff
@@ -398,6 +403,7 @@ public void shouldStoreOnlyEffectiveACEs() throws Exception {
     }
 
     @Test
+    @Deploy("org.nuxeo.elasticsearch.core.test:dummy-bulk-login-config.xml")
```

Comment:
You can use the one present in `nuxeo-core-bulk`test-jar.

Reply:
Actually, the one in nuxeo-core-bulk test jar is already loaded but override by another module, it has to be deployed there, at the last moment, with another component id